cmd/atc-installer/installer/run.go

```diff
@@ -209,18 +210,27 @@ func Run(cfg Config) (flight.Resources, error) {
 							Name:            "yokecd-atc",
 							Image:           cmp.Or(cfg.Image, "ghcr.io/yokecd/atc") + ":" + cfg.Version,
 							ImagePullPolicy: cmp.Or(cfg.ImagePullPolicy, corev1.PullIfNotPresent),
-							Env: []corev1.EnvVar{
-								{Name: "PORT", Value: strconv.Itoa(cfg.Port)},
-								{Name: "TLS_CA_CERT", Value: "/conf/tls/ca.crt"},
-								{Name: "TLS_SERVER_CERT", Value: "/conf/tls/server.crt"},
-								{Name: "TLS_SERVER_KEY", Value: "/conf/tls/server.key"},
-								{Name: "SVC_NAME", Value: svc.Name},
-								{Name: "SVC_NAMESPACE", Value: svc.Namespace},
-								{Name: "SVC_PORT", Value: strconv.Itoa(int(svc.Spec.Ports[0].Port))},
-								{Name: "DOCKER_CONFIG_SECRET_NAME", Value: cfg.DockerConfigSecretName},
-								{Name: "LOG_FORMAT", Value: cfg.LogFormat},
-								{Name: "VERBOSE", Value: strconv.FormatBool(cfg.Verbose)},
-							},
+							Env: func() []corev1.EnvVar {
+								env := []corev1.EnvVar{
+									{Name: "PORT", Value: strconv.Itoa(cfg.Port)},
+									{Name: "TLS_CA_CERT", Value: "/conf/tls/ca.crt"},
+									{Name: "TLS_SERVER_CERT", Value: "/conf/tls/server.crt"},
+									{Name: "TLS_SERVER_KEY", Value: "/conf/tls/server.key"},
+									{Name: "SVC_NAME", Value: svc.Name},
+									{Name: "SVC_NAMESPACE", Value: svc.Namespace},
+									{Name: "SVC_PORT", Value: strconv.Itoa(int(svc.Spec.Ports[0].Port))},
+									{Name: "DOCKER_CONFIG_SECRET_NAME", Value: cfg.DockerConfigSecretName},
+									{Name: "LOG_FORMAT", Value: cfg.LogFormat},
+									{Name: "VERBOSE", Value: strconv.FormatBool(cfg.Verbose)},
+								}
+								if cfg.ValidationWebhookTimeout != nil {
+									env = append(env, corev1.EnvVar{
+										Name:  "VALIDATION_WEBHOOK_TIMEOUT",
+										Value: strconv.FormatInt(int64(*cfg.ValidationWebhookTimeout), 10),
```

Comment:
if we use an `int` which is completely reasonable as the installer does not need to be optimized (its run once or once in a while) we could use the `strconv.Itoa` and avoid the type casts and dereferencing.